runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/renderers/GatewayFormRenderer.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { ChangeEvent } from 'react';
+
+import { Box, FormControl, Grid, InputLabel, MenuItem, Select, TextField, Typography } from '@mui/material';
+
+import { useModelerContext } from '#src-app/hooks/useModelerContext';
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import { useSelector } from '#src-app/store';
+
+import { ExpressionTextField, FlowExpression, GatewayFormMenu } from './GatewayFormRenderer.styles';
+
+import { BpmnConnectionFactory, IBpmnConnection, IBpmnGateway } from '../../helpers/elementParameters';
+
+import FlowLabelForm from '../FlowLabelForm';
+
+
+const GatewayFormRenderer = () => {
+    const COLOR_RB = '#FBB040';
+    const COLOR_BLACK = '#000000';
```

Comment:
isnt it available in our theme?

Reply:
imo use useTheme hook to get it

Reply:
default theme object is here https://mui.com/material-ui/customization/default-theme/